model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: updowncounter
+    unit: "{change}"
+    stability: experimental
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.url.full
+        requirement_level: required
+  - id: metric.vcs.repository.change.time
+    type: metric
+    metric_name: vcs.repository.change.time
+    brief: 'The time distribution a change (pull request) has been in open state or that it took to go from open to another state.'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.change.state
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_approval
+    type: metric
+    metric_name: vcs.repository.change.time_to_approval
+    brief: 'The amount of time it took a change (pull request) to go from open to approved'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.count
+    type: metric
+    metric_name: vcs.repository.count
+    brief: 'The number of repositories in an organization'
+    instrument: gauge
+    unit: "{repository}"
+    stability: experimental
+    attributes: []
+  - id: metric.vcs.repository.ref.count
+    type: metric
+    metric_name: vcs.repository.ref.count
+    brief: 'The number of refs of type branch or tag in a repository'
+    instrument: updowncounter
+    unit: "{ref}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.ref.lines_added
+    type: metric
+    metric_name: vcs.repository.ref.lines_added
```

Comment:
should this be a ref metric? I'm definitely not an expert, but I think it's more common to measure number of lines added/removed/changed in a PR, not between branches.